src/common/http-exception.ts

```diff
@@ -0,0 +1,16 @@
+// src/common/http-exception.ts
+
```

Comment:
Please document the code, explaining what each variable represents and the results obtained using JSDoc standard.

https://gomakethings.com/whats-the-best-way-to-document-javascript/